keyboards/macroflow_original/keymaps/default/keymap.c

```diff
@@ -0,0 +1,297 @@
+// Copyright 2024 Patrick Mathern (@Patrickemm)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+// Define the keycode, `QK_USER` avoids collisions with existing keycodes
+enum keycodes {
+  KC_CYCLE_LAYERS = QK_USER,
+};
+
+enum layer_names {
+  one,
+  two,
+  three,
+  four,
+  five,
+  six,
+  seven,
+  eight
+};
```

Comment:
This isn't used.
```suggestion
```

Reply:
They were used within the OLED render code, and its removal will require other changes.